src/avlizer_confluent.erl

```diff
@@ -311,8 +357,13 @@ get_registry_url() ->
 download(Ref) ->
   gen_server:call(?SERVER, {download, Ref}, infinity).
 
+%% Download schema from registry and insert into cache.
```

Comment:
Missing `@doc` or `@private` tag

Reply:
Added @doc tag, though it's a private function.